.github/workflows/ci.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
           # TODO: intentionally fails
           # - 'eager-fetch'
           - 'nestjs'
+          - 'next-app'
```

Comment:
nit: why is "-app" a suffix here? doesn't seem consistent with the others

Reply:
factory tool was called `create-next-app` 🤷  https://nextjs.org/docs/api-reference/create-next-app